app/code/Magento/Sitemap/Model/SitemapItem.php

```diff
@@ -90,4 +90,17 @@ public function getUpdatedAt()
     {
         return $this->updatedAt;
     }
+
+    public function toArray()
```

Comment:
Please, clarify the purpose of this method, looks like it never used

Reply:
Removed